packages/roosterjs-editor-dom/lib/list/VListItem.ts

```diff
@@ -20,9 +21,20 @@ import {
 import type { CompatibleListType } from 'roosterjs-editor-types/lib/compatibleTypes';
 
 const orderListStyles = [null, 'lower-alpha', 'lower-roman'];
+const unorderedListStyles = ['disc', 'circle', 'square'];
+const maxEnum =
+    BulletListType.Max > NumberingListType.Max ? BulletListType.Max : NumberingListType.Max;
+
+const numberDefinition = createNumberDefinition(
```

Comment:
you just need to keep this one and export it, and use it from VList. No need to have two separate definitions. Not only for code size, but also they need to always be the same. If some day we change one of them for some reason and forget to change the other one, there will be bug

Reply:
or, get/set the metadata always from a same file, don't split them.